TestingTool/tests/dsc_correctdata.json

```diff
@@ -0,0 +1,154 @@
+{
+    "name": "Validate the correct data on DSC",
+    "url": "",
+    "mobile_mode": "true",
+    "tasks": [
+        {
+            "name": "Go to DSC page",
+            "action": "go_to_url",
+            "args": {
+                "url":"http://79.112.76.0:32774"
+            }
+        },
+        {
+            "name": "Press Register Button ",
+            "action": "click_element",
+            "args": {
+                "xpath": "/html/body/app-root/div/app-welcome-page/div/div[2]/button[1]/span"
+            }
+        },
+        {
+            "name": "Press First name bar ",
+            "action": "click_element",
+            "args": {
+                "xpath": "//*[@id=\"mat-input-0\"]"
+            }
+        },
+        {
+            "name": "Insert first name",
+            "action": "input_element",
+            "args": {
+                "xpath": "//*[@id=\"mat-input-0\"]",
+                "input_text": "Adrian "
+            }
+        },
+
+        {  "name": "Check if error message appears",
```

Comment:
Trebuie verificat daca apare eroare la camp dupa ce ai dat click in afara lui, altfel nu ar aparea eroarea.